src/ui/public/angular-bootstrap/datepicker/month.html

```diff
@@ -1,15 +1,18 @@
 <table role="grid" aria-labelledby="{{uniqueId}}-title" aria-activedescendant="{{activeDateId}}">
   <thead>
     <tr>
-      <th><button type="button" class="btn btn-default btn-sm pull-left" ng-click="move(-1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-left"></i></button></th>
-      <th><button id="{{uniqueId}}-title" role="heading" aria-live="assertive" aria-atomic="true" type="button" class="btn btn-default btn-sm" ng-click="toggleMode()" tabindex="-1" style="width:100%;"><strong>{{title}}</strong></button></th>
-      <th><button type="button" class="btn btn-default btn-sm pull-right" ng-click="move(1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-right"></i></button></th>
+      <th><button type="button" class="kuiButton kuiButton--basic kuiButton--small pull-left" ng-click="move(-1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-left"></i></button></th>
```

Comment:
Before:
<img width="324" alt="screen shot 2017-04-21 at 9 16 09 am" src="https://cloud.githubusercontent.com/assets/16563603/25279037/3108bf70-2673-11e7-98e6-012a13126d2a.png">


After:
<img width="310" alt="screen shot 2017-04-21 at 9 10 33 am" src="https://cloud.githubusercontent.com/assets/16563603/25278953/e8be6878-2672-11e7-8a5f-6d4a314f9489.png">
The blue outline is cut off at the bottom, the hollow one might be too, hard to tell, and the background color doesn't quite match the background of the pane.

Might need to adjust the focus styling:
<img width="96" alt="screen shot 2017-04-21 at 9 16 58 am" src="https://cloud.githubusercontent.com/assets/16563603/25279062/4fb7c614-2673-11e7-9cf4-a26a1b584b43.png">